requirements/requirements.txt

```diff
@@ -530,8 +530,6 @@ yarl==1.18.3
     # via aiohttp
 zipp==3.21.0
     # via importlib-metadata
-zope-interface==7.2
-    # via twisted
```

Comment:
I don't know that we can do this. This is an indirect dependency, which is why we have the `./updater.sh` script to run. Twisted is an explicit requirement still.

https://github.com/ansible/awx/blob/d3ee9a1bfd0c927ab32756cfb203303259c503d6/requirements/requirements.in#L62

That pin was probably put in place for a CVE (so it can be removed), but it may still be required by other packages.

https://github.com/ansible/awx/blob/d3ee9a1bfd0c927ab32756cfb203303259c503d6/requirements/requirements.txt#L492-L495

And it seems that is `daphne`. We're on 4.1.2 for that. Today that still lists twisted as a dependency.

https://github.com/django/daphne/blob/main/pyproject.toml#L27

Reply:
@AlanCoding - Thanks for catching this.. I thought tests would catch it. I put this back in, which makes me think I need to revert the license files associated with it as well.

Reply:
Well now you're kind of left with nothing. But it's possible that upgrading zope could resolve the warning. Or maybe it has already been resolved with a prior update. And we should be able to remove the twisted pin in requirements.in, since we would never _downgrade_ our dependencies for no reason.